test/utils.ts

```diff
@@ -258,3 +263,75 @@ export function getLocalEip712Signature(
     // @ts-ignore
     return sigUtil.signTypedData_v4(privateKey, { data: typedRequestData });
 }
+
+export async function deployHub(
+    penalizer: string = constants.ZERO_ADDRESS,
+    configOverride: Partial<RelayHubConfiguration> = {}
+): Promise<RelayHubInstance> {
+    const relayHubConfiguration: RelayHubConfiguration = {
+        ...defaultEnvironment.relayHubConfiguration,
+        ...configOverride
+    };
+    return await RelayHub.new(
+        penalizer,
+        relayHubConfiguration.maxWorkerCount,
+        relayHubConfiguration.minimumEntryDepositValue,
+        relayHubConfiguration.minimumUnstakeDelay,
+        relayHubConfiguration.minimumStake
+    );
+}
+
+export async function getGaslessAccount(): Promise<AccountKeypair> {
+    const a = ethWallet.generate();
+    const gaslessAccount = {
+        privateKey: a.getPrivateKey(),
+        // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+        address: bufferToHex(privateToAddress(a.getPrivateKey())).toLowerCase()
+    };
+
+    return gaslessAccount;
+}
+
+export async function evmMineMany(count: number): Promise<void> {
+    for (let i = 0; i < count; i++) {
+        await evmMine();
+    }
+}
+
+export async function evmMine(): Promise<any> {
+    return await new Promise((resolve, reject) => {
+        // @ts-ignore
+        web3.currentProvider.send(
+            {
+                jsonrpc: '2.0',
+                method: 'evm_mine',
+                params: [],
+                id: Date.now()
+            },
+            (e: Error | null, r: any) => {
+                if (e) {
+                    reject(e);
+                } else {
+                    resolve(r);
+                }
+            }
+        );
+    });
+}
+
+// encode revert reason string as a byte error returned by revert(stirng)
+export function encodeRevertReason(reason: string): PrefixedHexString {
```

Comment:
don't seem to be able to find any usage of this method

Reply:
Removed